packages/playwright-test/types/testReporter.d.ts

```diff
@@ -20,15 +20,15 @@ export type { FullConfig, TestStatus } from './test';
 
 /**
  * `Suite` is a group of tests. All tests in Playwright Test form the following hierarchy:
- * - Root suite has a child suite for each [TestProject].
+ * - Root suite has a child suite for each [TestProject](https://playwright.dev/docs/api/class-testproject).
```

Comment:
Perhaps this should be [`@link`](https://github.com/microsoft/vscode-docs/blob/1735dfae164f4593912182843fd04cfb97c9f69a/release-notes/v1_57.md#jsdoc-link-support) instead?